.github/workflows/codeql-analysis.yml

```diff
@@ -27,16 +27,13 @@ jobs:
       contents: read
       security-events: write
 
-    strategy:
-      fail-fast: false
```

Comment:
Mmmh it's useful sometimes to have all results every time. We can quickly see if it's a device specific failure or global failure
Especially if we have some nanos around